docs/docs-ref-autogen/excel/excelscript/excelscript.worksheet.yml

```diff
@@ -160,7 +160,50 @@ methods:
           type: string
       return:
         type: '<xref uid="ExcelScript!ExcelScript.Shape:interface" />'
-        description: ''
+        description: |-
+
+
+          #### Examples
+
+          ```typescript
+          /**
+           * This sample copies an image from a URL. This could be used to copy photos that a colleague stored in a shared folder to a related workbook.
+           */
+          async function main(workbook: ExcelScript.Workbook) {
+            // Fetch the image from a URL.
+            const link = "https://github.com/OfficeDev/office-scripts-docs/blob/master/docs/images/git-octocat.png";
+            const response = await fetch(link);
+
+            // Store the response as an ArrayBuffer, since it is a raw image file.
+            const data = await response.arrayBuffer();
+
+            // Convert the image data into a base64-encoded string.
+            const image = convertToBase64(data);
+
+            // Add the image to a worksheet.
+            workbook.getWorksheet("WebSheet").addImage(image)
```

Comment:
Is there a way to get the 'current' worksheet? I had to add this sheet name to my Excel file for this to work.